### PR TITLE
Thrown mobs cant move until the throw ends

### PR DESCRIPTION
**Pull request by HMBGERDO**
## What Does This PR Do

This PR blocks moving if you're currently being thrown. 

## Why It's Good For The Game

If you're being thrown, being able to stop throw by any movement button is... weird. 
Xeno hunter stopping jump in the middle of leap? Shadow demon being able to not move after using grapple hand on non living target? Gargantua failing charge spell because you pressed W in fight? not like this. 
Being thrown is not easy and being able to just press button to make it much lesser than it supposed to be  or to completely kill spell based on throwing is weird.

## Testing

Compiled, tested various throwing

## Changelog
:cl:
tweak: You cant move while being thrown
/:cl:

### CODE REVIEW

**Comment by Qwertytoforty** [comment]
Could you make it argument on throws perhaps? I can see some abilities, such as the modsuit grappling hook, being a real pain to use when trying to attach to walls in space.

That and a person throwing you in space locking your movement forever even if you have a jetpack and it loops, could be an issue

**Comment by HMBGERDO** [comment]
if the place has no gravity, throw will end instantly meaning you can move
i will think about throws with move allowed, wait for updates

**Comment by HMBGERDO** [comment]
Alright, we can choose which throws will block movement now. Added tether engineering modsuit module to not blocking movement throw(though it didnt block even before because of space and no gravity) but still

**Review by Sirryan2002 — APPROVED** [review]

File: code/controllers/subsystem/SSthrowing.dm [hunk 1]
```diff
@@ -70,6 +70,8 @@ SUBSYSTEM_DEF(throwing)
 	var/last_move = 0
 	///When this variable is false, non dense mobs will be hit by a thrown item. useful for things that you dont want to be cheesed by crawling, EG. gravitational anomalies
 	var/dodgeable = TRUE
+	///Should thrownthing movement be blocked if its a mob?
```
Comment (lewcc) [inline code/controllers/subsystem/SSthrowing.dm:73]: Slightly clearer wording:
```suggestion
	/// Can a thrown mob move themselves to stop the throw?
```

**Review by lewcc — APPROVED** [review]

**Pull request merged**